install/kubernetes/cilium/values.yaml.tmpl

```
@@ -1335,6 +1335,18 @@ monitor:
   # endpoints filtering
   # serviceTopology: false
 
+  # -- LoadBalancer with envoy proxy
+  envoy:
+    # -- Enable L7 service load balancing via envoy proxy.
+    # The request to a k8s service, which has specific annotation e.g. io.cilium/lb-protocol,
+    # will be forwarded to the local envoy proxy to be load balanced to the service endpoints.
+    # Please refer to docs for supported annotations for configuring the envoy proxy.
+    # This will automatically set enable-envoy-config as well.
+    enabled: false
+    # -- List of ports from service to be automatically redirected to envoy proxy.
+    # Fine-grained control can be achieved by using the service annotation.
+    ports: [ ]
```

nit (to be consistent with the rest of the file):
```suggestion
    ports: []
```